### DISCUSSION TITLE
How to render multiple qmd files from a default project into one output file?

### DISCUSSION THREAD

**Discussion by i-walker** [quarto-dev/quarto-cli#9348] [Q&A] [answered]
### Description

essentially I have:

project
- _quarto.yaml
- /subdir
  - page-one.qmd
  - page-two.qmd
  - page-three.qmd
-/moresubdir
 
And I hope to create a metadata.yaml file, where I can specify the order how the files are merged into one PDF, when I hit render on the subdirectories. Without using project - type book, since my other subdirectories need other specs.

- What Quarto version are you using?
1.4.553

- What operating system are you using?
macOS 14.4

**Comment by mcanouil** [comment, marked as answer]
I don't know what you mean by "combine" and what your different documents look like.

Book project is the only thing allowing independent document to be merged (one set of metadata).

You could use the include shortcode (see documentation) or use a post-render script to append the PDF to one another.

Regarding how metadata are shared, see <https://quarto.org/docs/projects/quarto-projects.html#rendering-projects>.

**Reply by i-walker** [reply]
Is it then possible to have a project set up of books that are merged per directory individually and copy the settings from the root metadata?

**Reply by mcanouil** [reply]
Sorry, but I don't understand your setup or what you are trying to do in the first place.
Could your clarify and possibly share a small concrete example?

Also you need to clarify what "merged" means to you.

FYI, in Quarto a "project" is a directory where there is `_quarto.yml`.
A "book" is a project type in Quarto.
"root metadata" are you referring to project level metadata which is `_quarto.yml`?

**Reply by i-walker** [reply]
Thanks I just used book as a type. With merged I just meant that several qmd files are added together
